CMake error on some systems

<!-- To report an issue, fill in all the applicable fields below. -->

#### Platform
Ubuntu 14.04 virtual machine

#### Steps to Reproduce
Clone repo and run `cmake ..`

#### Observed result
> CMake Error at cmake/WaveUtils.cmake:16 (CMAKE_PARSE_ARGUMENTS):
  Unknown CMake command "CMAKE_PARSE_ARGUMENTS".

#### Possible cause
CMakeParseArguments module must be included
(https://cmake.org/Bug/bug_relationship_graph.php?bug_id=14230&graph=dependency).
That it worked on other machines is just luck.

--- a/cmake/WaveUtils.cmake
+++ b/cmake/WaveUtils.cmake
@@ -1,6 +1,6 @@
 # - Macros used in libwave
 
-include(CMakeParseArguments)
+INCLUDE(CMakeParseArguments)
 
 # wave_add_test: Add a gtest target
 #

--- a/cmake/WaveUtils.cmake
+++ b/cmake/WaveUtils.cmake
@@ -1,5 +1,7 @@
 # - Macros used in libwave
-#
+
+include(CMakeParseArguments)
+
 # wave_add_test: Add a gtest target
 #
 # WAVE_ADD_TEST(Name [DISABLED] src1 [src2...])

